packages/libs/error-stack/Makefile.toml

```diff
@@ -107,13 +107,13 @@ private = true
 run_task = { name = ['update-snapshots-task-lib', 'update-snapshots-task-doc'] }
 
 [tasks.update-snapshots-task-lib]
-extend = "task"
+extend = "cargo"
```

Comment:
Should I cherry-pick these (as in the respective bits across the whole PR) over to main? @TimDiekmann

Reply:
I think that makes sense, yes 🙂